plugins/xpad/xpad.cpp

```diff
@@ -23,6 +23,10 @@
 #include "xpad.h"
 #include <VersionHelpers.h>
 
+#undef _WIN32_WINNT
+#define _WIN32_WINNT 0x0602 // Required for XInputEnable definition
```

Comment:
> it simply allows me to use decltype for function pointers

Ohh, so that was the point, ok. Couldn't it be a compiler [bug](https://connect.microsoft.com/VisualStudio/Feedback/Details/2173340)?